ods_ci/tests/Tests/100__deploy/130__operators/130__rhods_operator/139__trusted_ca_bundles.robot

```diff
@@ -73,10 +73,11 @@ Validate Trusted CA Bundles State Removed
 *** Keywords ***
 Suite Setup
     [Documentation]    Suite Setup
+    Assign Vars According To Product    ${PRODUCT}
```

Comment:
This is not needed, as it will be already called in RHOSI Setup

Reply:
I had to include it as an input param because of:
`Validate Trusted CA Bundles State Managed :: The purpose of this t... | FAIL |
Parent suite setup failed:
Keyword 'RHOSi.Assign Vars According To Product' expected 1 argument, got 0.`